src/sentry/static/sentry/app/views/ruleEditor/ruleNode.jsx

```diff
@@ -40,11 +43,13 @@ class RuleNode extends React.Component {
 
   render() {
     let {data, node} = this.props;
+    let html = utils.getHtmlForNode(node);
```

Comment:
* Is it possible to use `GenericField` here?
* How much of a change would ^ involve?

Reply:
@billyvg That component is very opinionated about how to manage your form data, I'd rather use simple components for now.